app/navigation/BorderNavigator.scala

```diff
@@ -44,20 +48,9 @@ class BorderNavigator @Inject() () extends Navigator {
 
   override def checkRoutes(departureId: String, mode: Mode): PartialFunction[Page, UserAnswers => Option[Call]] = ???
 
-  private def borderModeNavigation(ua: UserAnswers, departureId: String, mode: Mode): Option[Call] = {
-    val numberOfActiveBorderMeans: Int = ua.get(BorderActiveListSection).map(_.value.length).getOrElse(0)
-
-    (ua.get(BorderModeOfTransportPage), ua.departureData.TransitOperation.security, ua.departureData.Consignment.ActiveBorderTransportMeans.isDefined) match {
-      //TODO: Change route for first case when page has been added
-      case (Some(BorderMode("5", _)), "0", true) =>
-        Some(controllers.routes.MoreInformationController.onPageLoad(departureId))
-      case _ => Some(routes.IdentificationController.onPageLoad(departureId, mode, Index(numberOfActiveBorderMeans)))
-    }
-  }
-
   private def customsOfficeNavigation(ua: UserAnswers, departureId: String, mode: Mode, activeIndex: Index): Option[Call] =
-    (ua.get(BorderModeOfTransportPage), ua.departureData.TransitOperation.security) match {
-      case (Some(BorderMode("4", _)), "1" | "2" | "3") =>
+    (ua.get(BorderModeOfTransportPage), ua.departureData.TransitOperation.isSecurityTypeInSet) match {
+      case (Some(BorderMode("4", _)), true) =>
```

Comment:
In config.Constants we have named values that can  be used instead of some of these numbers eg   val Air      = "4"